### PR TITLE
Update Honzon benchmark to use worst case swap

### PR DESCRIPTION
**Pull request by ferrell-code**
Todo: Add benchmarks for case where `T::Swap` goes through taiga instead of the DEX, may be easier after https://github.com/AcalaNetwork/Acala/pull/2237/ is merged

### CODE REVIEW

File: runtime/mandala/src/lib.rs [hunk 1]
```diff
@@ -1146,7 +1146,7 @@ parameter_types! {
 	];
 	pub const ExtendedProvisioningBlocks: BlockNumber = 2 * DAYS;
 	pub const TradingPathLimit: u32 = 4;
-	pub AlternativeSwapPathJointList: Vec<Vec<CurrencyId>> = vec![vec![GetStakingCurrencyId::get()]];
+	pub AlternativeSwapPathJointList: Vec<Vec<CurrencyId>> = vec![vec![GetStakingCurrencyId::get()], vec![GetLiquidCurrencyId::get()]];
```
Comment (ferrell-code) [inline runtime/mandala/src/lib.rs:1149]: Could feature gate the benchmarks instead, but figured mandala mirroring acala/karura is better anyways

**Review by xlc — DISMISSED** [review]

**Review by xlc — DISMISSED** [review]

**Review by xlc — APPROVED** [review]

**Pull request merged**